setup.py

```diff
@@ -17,7 +17,9 @@
     'requests',
     'jsonpickle',
     'pyelliptic',
-    'ecdsa']
+    'ecdsa',
+    'collective.indexing == 2.0b1'
```

Comment:
Better to keep versions in requirements.txt so that they are easier to bump by [bump](https://pypi.python.org/pypi/bumper/0.1.8)